### DISCUSSION TITLE
Dependabot parser treat hours as integers

### DISCUSSION THREAD

**Discussion by mondeja** [community/community#26363] [Actions]
<p>I want to use <code>time: 04:00</code> in my Dependabot configuration files, but I must use quoted strings like <code>time: '04:00'</code> or I will get an error like <a href="https://github.com/mondeja/latex-ji18n/runs/1459035577" rel="noopener nofollow ugc">this</a>: <code>The property '#/updates/1/schedule/time' of type integer did not match the following type: string</code>.</p>
<p>What parser is using dependabot? Every parser that I’ve tested parses hours as strings, but dependabot not :thinking:</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/mondeja">@mondeja</a>, welcome to the GitHub Support Community! Dependabot’s configuration is in YAML and <code>04:00</code> is a valid sexagesimal integer in YAML. You need to explicitly define the <code>time</code> variable as a string type using <code>"</code> or <code>'</code> in order for the parser to be able to read this properly.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#scheduletime" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="16" height="16">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#scheduletime" target="_blank" rel="noopener">Configuration options for dependency updates - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by mondeja** [comment]
<p>Oh, I haven’t noticed but <a href="https://yaml.org/type/int.html" rel="noopener nofollow ugc">it’s well defined in the YAML spec</a>. Thank you! :+1:</p>
